lib/buffer.txt

```diff
@@ -0,0 +1,216 @@
+class PersonalForm extends StatefulWidget {
+  const PersonalForm({super.key});
+
+  @override
+  State<PersonalForm> createState() => _PersonalFormState();
+}
+
+class _PersonalFormState extends State<PersonalForm> {
+  @override
+  void initState() {
+    super.initState();
+
+    _firstNameController = TextEditingController();
+    _lastNameController = TextEditingController();
+  }
+
+  @override
+  void dispose() {
+    _firstNameController.dispose();
+    _lastNameController.dispose();
+
+    super.dispose();
+  }
+
+  late TextEditingController _firstNameController;
```

Comment:
these should be late final TextEditingController since the controller is never updated